tangorb/dao/src/main/java/fr/esrf/TangoApi/events/EventConsumer.java

```diff
@@ -53,13 +53,13 @@
         implements TangoConst, IEventConsumer {
 
     protected static int subscribe_event_id = 0;
-    protected static Hashtable<String, EventChannelStruct>   channel_map        = new Hashtable<String, EventChannelStruct>();
-    protected static Hashtable<String, String>               device_channel_map = new Hashtable<String, String>();
-    protected static Hashtable<String, EventCallBackStruct>  event_callback_map = new Hashtable<String, EventCallBackStruct>();
-    protected static Hashtable<String, EventCallBackStruct>  failed_event_callback_map = new Hashtable<String, EventCallBackStruct>();
+    protected static Hashtable<String, EventChannelStruct>   channel_map        = new Hashtable<>();
```

Comment:
Yes.
The same maps are used by ZMQ EventConsumer and Notifd EventConsumer.
It has been developed as static 12 years ago, why do you want to change now ?